app/controllers/supervisor/base_controller.rb

```diff
@@ -0,0 +1,18 @@
+class Supervisor::BaseController < ApplicationController
```

Comment:
I followed the existing BaseControllers - apart from the fact that I wasn't aware that I was adding a Module ;-)  
So the question is: what is more important: Consistency between the Base Controllers, or follow this best practice? Or, @carpodaster: I _could_ change all of them?